jerry-main/main-unix.c

```diff
@@ -627,7 +627,7 @@ main (int argc,
 
   if (jerry_is_feature_enabled (JERRY_FEATURE_SNAPSHOT_EXEC))
   {
-    for (int i = 0; i < exec_snapshots_count; i++)
+    for (i = 0; i < exec_snapshots_count; i++)
```

Comment:
I thinks won't be right. You will override the `i` of the outer `for` statement here. This is unsafe and might lead to infinite loop. Please rename the variable instead, eg.: `for (int j = 0; j < exec_snapshots_count; j++)`